com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs

```diff
@@ -68,9 +68,18 @@ public static class CoreEditorStyles
         /// <summary>Help icon style</summary>
         public static GUIStyle iconHelpStyle => GUI.skin.FindStyle("IconButton") ?? EditorGUIUtility.GetBuiltinSkin(EditorSkin.Inspector).FindStyle("IconButton");
 
+        /// <summary>Style of Section Headers.</summary>
+        public static GUIStyle sectionHeaderStyle = new GUIStyle(EditorStyles.largeLabel) { richText = true, fontSize = 18, fixedHeight = 42 };
+        /// <summary>Style of Sub-Section Headers.</summary>
+        public static GUIStyle subSectionHeaderStyle = new GUIStyle(EditorStyles.boldLabel);
```

Comment:
For those, it should be available in trunk in order to always share same style accross the product.
(At the same time, it is the case for almost everything in this class)

Reply:
yes i agree. not sure how to solve this though.

Reply:
Can we add them on EditorStyles?

Reply:
well who has the ownership of that?

Reply:
The Editor Team? Who are the people that commit to this file?